controllers/authpolicy_status.go

```diff
@@ -27,7 +27,7 @@ func (r *AuthPolicyReconciler) reconcileStatus(ctx context.Context, ap *kuadrant
 	if specErr == nil { // skip fetching authconfig if we already have a reconciliation error.
 		apKey := client.ObjectKeyFromObject(ap)
 		authConfigKey := client.ObjectKey{
-			Namespace: kuadrantNamespace,
+			Namespace: ap.Namespace,
```

Comment:
So I reckon it wasn't needed to place the AuthConfigs where Authorino would be (Kuadrant NS)... then there's no need to annotate the AuthPolicy nor get Kuadrant namespace which explains why the logic was removed and simplified :)